package.json

```
@@ -13,7 +13,9 @@
     "npm": ">=8.3.0"
   },
   "overrides": {
-    "ansi-regex": "3.0.1"
+    "ansi-regex": "3.0.1",
+    "json5": "2.2.2",
```

```suggestion
    "json5": "2.2.3",
```

If we bump this to 2.2.3 we could close https://github.com/Stedi/ts2asl/pull/518 as well